tst/functional/TestKeepsGoingUp.kt

```diff
@@ -73,12 +82,40 @@ class TestKeepsGoingUp {
         val map = builder.build()
 
         val player = GeneralPlayer(actor)
-        val game = Game(player, map)
+        val condition: (Move) -> InvalidMoveException? =
```

Comment:
why is this defined and repeated for every test shouldn't this be a part of game engine?